doc/source/ray-air/examples/serving_guide.ipynb

```diff
@@ -0,0 +1,923 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(air-serving-guide)=\n",
```

Comment:
On the GitHub render, this line seems to show up as:
 
<img width="1224" alt="Screen Shot 2022-06-03 at 1 46 40 PM" src="https://user-images.githubusercontent.com/92341594/171950129-7c1622fc-ca4a-46ae-820a-7ee30fb16b74.png">
 
Can we hide that somehow?

Reply:
This is intentional and it will be gone for actual rendering in HTML